pkg/apis/ceph.rook.io/v1/network_test.go

```diff
@@ -48,6 +48,24 @@ func TestNetworkCephIsHostLegacy(t *testing.T) {
 	assert.True(t, net.IsHost())
 }
 
+func TestNetworkCephIsHost(t *testing.T) {
+	net := NetworkSpec{Provider: "host"}
+
+	assert.True(t, net.IsHost())
+	net = NetworkSpec{Provider: "host-strict"}
+
+	assert.True(t, net.IsHost())
```

Comment:
It's best visually to keep test setup and test check together. 

```suggestion
	net := NetworkSpec{Provider: "host"}
        assert.True(t, net.IsHost())
	
	net = NetworkSpec{Provider: "host-strict"}
	assert.True(t, net.IsHost())
```